### PR TITLE
Add 1.24 interview

### PR DESCRIPTION
**Pull request by craigbox**
Publish an abridged transcript of the Kubernetes Podcast interview with James Laverack as a kubernetes.io blog post. 

Tradition is that this goes live the week before the next release.

See also: https://github.com/kubernetes/website/pull/33280, https://github.com/kubernetes/website/pull/30688, https://github.com/kubernetes/website/pull/29146, https://github.com/kubernetes/website/pull/22882, https://github.com/kubernetes/website/pull/22716, https://github.com/kubernetes/website/pull/17963 et al.

/cc @JamesLaverack 
/cc @sftim

### CODE REVIEW

**Comment by craigbox** [comment]
Preview at https://deploy-preview-36051--kubernetes-io-main-staging.netlify.app/blog/2022/08/18/stargazing-solutions-and-staycations-the-kubernetes-1.24-release-interview/.

File: content/en/blog/_posts/2022-08-18-kubernetes-1.24-release-interview.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+
+JAMES LAVERACK: Ahh, annoyed, because she was right. This was kind of a couple of months ago when I was in the middle of it with all of the Kubernetes meetings. Also, my client project at the time involved a lot of technical discussion. I was in three or four hours of calls every day. And I don't mind that. But I would come out, in part because of course you're working from home, so she sees me all the time. So I'd come out, I'd grab a coffee and be like, "oh, I've got a meeting, I've got to go." And she'd be like, "do you ever code anymore?"
+I think it was in fact just after Christmas when she asked me, "when was the last time you programmed anything?" And I had to think about it. Then I realized that perhaps there was a problem there. Well, not a problem, but I realized that perhaps I don't code as much as I used to.
+
+**CRAIG BOX: Are you the kind of person who will pick up a hobby project to try and fix that?**
+
+JAMES LAVERACK: Absolutely. I've recently started writing [a Kubernetes operator for my Minecraft server]](https://github.com/JamesLaverack/minecraft-operator). That probably tells you about the state I'm in.
```
Comment (reylejano) [inline content/en/blog/_posts/2022-08-18-kubernetes-1.24-release-interview.md:102]: link didn't render properly on the preview
```suggestion
JAMES LAVERACK: Absolutely. I've recently started writing [a Kubernetes operator for my Minecraft server](https://github.com/JamesLaverack/minecraft-operator). That probably tells you about the state I'm in.
```

File: content/en/blog/_posts/2022-08-18-kubernetes-1.24-release-interview.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+
+**CRAIG BOX: You've been a member of every release team since that point, since Kubernetes 1.18 in 2020, in a wide range of roles. What shared roles have you been through?**
+
+JAMES LAVERACK: I started out as a release notes shadow, and did that for a couple of releases, in 1.18 and 1.19. In 1.20, I was the release notes lead. And then in 1.21, I moved into being a shadow again as an enhancement shadow, before in 1.22 becoming an enhancements lead, but in 1.23 a release lead shadow, and finally in 1.24, release lead as a whole.
+
+**CRAIG BOX: That's quite a long time to be with the release team. You're obviously going to move into an emeritus role after this release. Do you see yourself still remaining involved? Is it something that you're clearly very passionate about?**
+
+JAMES LAVERACK: I think I'm going to be around in SIG Release for as long as people want me there. I find it a really interesting part of the community. And I find the people super-interesting and super-inviting.
+
+**CRAIG BOX: Let's talk then about [Kubernetes 1.24](https://kubernetes.io/blog/2022/05/03/kubernetes-1-24-release-announcement/). First, as always, congratulations on the release.**
```
Comment (reylejano) [inline content/en/blog/_posts/2022-08-18-kubernetes-1.24-release-interview.md:154]:
```suggestion
**CRAIG BOX: Let's talk then about [Kubernetes 1.24](/blog/2022/05/03/kubernetes-1-24-release-announcement/). First, as always, congratulations on the release.**
```

**Comment by reylejano** [comment]
/lgtm
@kubernetes/sig-docs-blog-owners , ptal

File: content/en/blog/_posts/2022-08-18-kubernetes-1.24-release-interview.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+
+**CRAIG BOX: Let's talk about some of the new features in 1.24. We have been hearing for many releases now about the impending doom which is the removal of Dockershim. [It is gone in 1.24](https://github.com/kubernetes/enhancements/issues/2221). Do we worry?**
+
+JAMES LAVERACK: I don't think we worry. This is something that the community has been preparing for for a long time. We've published a lot of documentation about how you need to approach this. The honest truth is that most users, most application developers in Kubernetes, will simply not notice a difference or have to worry about it.
+
+It's only really platform teams that administer Kubernetes clusters and people in very specific circumstances that are using Docker directly, not through the Kubernetes API, that are going to experience any issue at all.
+
+**CRAIG BOX: And I see that Marantis and Docker have developed a CRI plugin for Docker anyway, so you can just switch over to that and everything continues.**
```
Comment (craigbox) [inline content/en/blog/_posts/2022-08-18-kubernetes-1.24-release-interview.md:176]:
```suggestion
**CRAIG BOX: And I see that Mirantis and Docker have developed a CRI plugin for Docker anyway, so you can just switch over to that and everything continues.**
```

**Comment by reylejano** [comment]
re-applying lgtm
/lgtm

File: content/en/blog/_posts/2022-08-18-kubernetes-1.24-release-interview.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+
+SLSA is a framework that describes software supply chain security. That is, of course, a really big issue in our industry at the moment. And it's really good to see the project adopting the best practices for this.
+
+**CRAIG BOX: I was looking back at [the conversation I had with Rey Lejano about the 1.23 release](https://kubernetespodcast.com/episode/167-kubernetes-1.23/), and we were basically approaching Level 2. We're now obviously stepping up to Level 3. I think I asked Rey at the time was, is it fair to say that SLSA is inspired by large projects like Kubernetes, and in theory, it should be really easy for these projects to tick the boxes to get to that level, because the SLSA framework is written with a project like Kubernetes in mind?**
+
+JAMES LAVERACK: I think so. I think it's been somewhat difficult, just because it's one thing to do it, but it's another thing to prove that you're doing it, which is the whole point around these frameworks — the ascertation, that proof.
```
[the remark this answers is not in the document]
Comment (craigbox) [reply content/en/blog/_posts/2022-08-18-kubernetes-1.24-release-interview.md]: Fixed most of those, missed that one!

**Comment by craigbox** [comment]
Thanks @JamesLaverack - ready to unhold?

**Comment by JamesLaverack** [comment]
/unhold
/lgtm

Thanks @craigbox!

**Comment by onlydole** [comment]
This looks good to me! Thank you, @craigbox and @JamesLaverack! 

/approve

**Pull request merged**